rbac/management/principal/view.py

```diff
@@ -111,7 +115,14 @@ def get(self, request):
         if offset - limit > 0:
             previous_offset = offset - limit
 
-        resp, usernames_filter = self.users_from_proxy(user, query_params, options, limit, offset)
+        # Attempt validating and obtaining the "prinicpal type" query
+        # parameter. For now, having "service-account" as the "principal type"
+        # returns a stub.
+        principalType = validate_principal_type(query_params)
```

Comment:
Here we could define the valid values and call `validate_and_get_key` as well?

Reply:
Yes, changed!

Reply:
Awesome!!